shared/settings/advanced/index.tsx

```diff
@@ -101,7 +101,7 @@ const Advanced = (props: Props) => {
           </Kb.Box>
         )}
         {isLinux ? <UseNativeFrame {...props} /> : null}
-        {!Styles.isMobile && !isLinux && (
+        {!Styles.isMobile && (
```

Comment:
Don't understand how props.openAtLogin is set on GUI start

Reply:
@heronhaye  In `actions/platform-specific/index.desktop` it is loaded and put in the store, then this component can get it.